test/integration/kubernetes_deploy_test.rb

```diff
@@ -794,7 +729,7 @@ def test_bad_container_on_stateful_sets_fails_with_rolling_update
       "StatefulSet/stateful-busybox: FAILED",
       "app: Crashing repeatedly (exit 1). See logs for more information.",
       "Events (common success events excluded):",
-      "[Pod/stateful-busybox-0]	FailedSync: Error syncing pod",
+      %r{\[Pod/stateful-busybox-\d\]	BackOff: Back-off restarting failed container},
```

Comment:
Always a single digit?

Reply:
SS pods are assigned a stable index based on the total number of replicas, so the test statefulset would need to have 11+ replicas for there to potentially be an additional digit